test/SourceKit/CodeComplete/complete_optionalmethod.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
```

Comment:
Is this really necessary in a test?  I don't think most of our tests with @objc are importing Foundation.